scripts/fns.sh

```
@@ -0,0 +1,14 @@
+# read last_md5_value
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)